styles.css

```diff
@@ -25,8 +24,7 @@
     flex-direction: column;
     gap: 8px;
     color: #18191F;
-    width: 540px;
-    height: 130px;
+    max-width: 90%;
```

Comment:
make it as exactly given in Figma ie. 540px. So, make it max-width: 540px. But, it would be better if you move this max-width: 540px to its parent class i.e. stats-section-firstcolumn.